docs/forms.md

```diff
@@ -0,0 +1,168 @@
+# Forms Migration Guide
+
+## Overview
+
+This application supports two form implementations.
+
+- Legacy: `FormWrapper` uses redux-form for state management. You'll see this in a lot of forms using things like 
+    `Field`, `getFormValues`, and `dispatch(change(...))`. `redux-form` is no longer maintained and we're moving away from it.
+- Replacement: `AntdFormWrapper` with `FormField` components. This uses `antd` form for form management.
+
+The AntdFormWrapper is built to keep the same API as the FormWrapper (as far as possible), to keep the migration process simple.
+
+## Migration Steps
+
+1. Replace wrapper components:
+   ```tsx
+   // Old
+   <FormWrapper
+     name="my_form"
+     onSubmit={handleSubmit}
+     reduxFormConfig={{touchOnChange: true}}
+   >
+
+   // New
+   const [form] = Form.useForm(); // This form instance is what can be used to replace things like getting access to form values, setting form values programmatically, etc.
+   <AntdFormWrapper
+     form={form} 
+     name="my_form"
+     onSubmit={handleSubmit}
```

Comment:
Might be nice to add a reference to `antdOptions` here as the alternative to the previous `reduxFormConfig`